x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -168,6 +178,48 @@ export function AlertDetails() {
   const AlertDetailsAppSection = ruleTypeModel ? ruleTypeModel.alertDetailsAppSection : null;
   const timeZone = getTimeZone(uiSettings);
 
+  const OVERVIEW_TAB_ID = 'overview';
+  const METADATA_TAB_ID = 'metadata';
+
+  const overviewTab = AlertDetailsAppSection && rule && alertDetail?.formatted && (
+    <>
+      <EuiSpacer size="l" />
+      <AlertSummary alertSummaryFields={summaryFields} />
+      <AlertDetailsAppSection
+        alert={alertDetail.formatted}
+        rule={rule}
+        timeZone={timeZone}
+        setAlertSummaryFields={setSummaryFields}
+        ruleLink={http.basePath.prepend(paths.observability.ruleDetails(rule.id))}
+      />
+    </>
+  );
+
+  const metadataTab = alertDetail?.raw && (
```

Comment:
Can we also have a unit test for `metadataTab` and check if one of the parameters that we passed is shown correctly there?

Reply:
updated in 1c25061873737610b24e7a141ec555517971ed52